### ISSUE TITLE
Update vulnerabilities

### ISSUE THREAD

**Issue by cgrimm013** [Picolab/Manifold#34]
update any packages that may be causing vulnerability issues.

**Comment by cgrimm013** [comment]
Not really sure what is up with this one, but I can't find the vulnerabilities in our dependency graph anymore. I assume that whatever package we rely on that was using it removed it from their dependency.

**Comment by burdettadam** [comment]
The package was hoek version < 3.

**Comment by cgrimm013** [comment]
I did remember that, but the problem is that I don't even see it in our dependency tree.

**Comment by cgrimm013** [comment]
Closing because there are no longer vulnerabilities.

**Comment by cgrimm013** [comment]
When Adam pushed his package-lock files, they included some outdated dependencies. The package-lock file is not our direct dependencies, but generally made up of our dependencies' dependencies.

**Comment by cgrimm013** [comment]
There is a new message (I think when I updated npm, but I didn't notice it at first) when installing new packages. Here is the message: 
![uploads/3a8273fb-99f5-4db0-a826-732ccb28362a/Screen Shot 2018-05-09 at 10.44.25 AM.png](https://s3-us-west-2.amazonaws.com/prod.huboard.com/uploads%2F3a8273fb-99f5-4db0-a826-732ccb28362a%2FScreen+Shot+2018-05-09+at+10.44.25+AM.png)

**Comment by cgrimm013** [comment]
After some more research, I found that we don't really need to worry about any of these issues. As npm receives reports from users about vulnerability issues, an npm audit will report those issues along with their severity. I found that all of our critical, high, and moderate severity vulnerabilities are in development dependencies and would not be provided in a production environment. Many of our low severity vulnerabilities are of the same cut, and the rest don't seem like our use cases will run into them from my shallow search. I'm closing this issue unless we figure out a need to do something more about this.
